padertorch/train/hooks.py

```diff
@@ -673,13 +673,13 @@ def _back_off(self, trainer: 'pt.Trainer'):
         latest_symlink_path.symlink_to(best_ckpt)
 
         best_iter = int(best_ckpt[len('ckpt_'): -len('.pth')])
-        for j in range(1, len(self.ckpt_ranking)):
-            ckpt = self.ckpt_ranking[-j][0]
+        for j in range(len(self.ckpt_ranking)-1, 0):
```

Comment:
Are you sure, that this does what you want?
```python
>>> l = [1, 2, 3]
>>> list(range(len(l)-1, 0))
[]
```

Reply:
Fixed it and added a test case